### DISCUSSION TITLE
Self sensor library from private git repository

### DISCUSSION THREAD

**Discussion by vitalerter** [zephyrproject-rtos/zephyr#44617] [Q&A]
**Is your feature request related to a problem? Please describe.**
I working with nrf sdk and have sensor that not exists at /sensor library.
I create a driver for this sensor but i can't bind it because it is not exists at zephyr\dts\bindings\sensor

**Describe the solution you'd like**
I want to be able tp bind any location of the sensor if it is meet the reequipments of implementation for example new implementation at external git repository(private) and link bind for it

**Comment by henrikbrixandersen** [comment]
Devicetree bindings can be located in the application directory. Please see https://docs.zephyrproject.org/latest/guides/dts/bindings.html#where-bindings-are-located
